assets/src/edit-story/components/library/panes/media/local/dropDownMenu.js

```diff
@@ -17,17 +17,17 @@
 /**
  * External dependencies
  */
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import PropTypes from 'prop-types';
-import { useCallback, useRef, useState } from 'react';
+import { useCallback, useRef, useState, useMemo } from 'react';
 import { __ } from '@web-stories-wp/i18n';
+import { v4 as uuidv4 } from 'uuid';
 
 /**
  * Internal dependencies
  */
+import { Menu, PLACEMENT, Popup } from '../../../../../../design-system';
```

Comment:
I suppose I see this menu here for images as a `contextMenu` - this could be done as a follow up, but it would probably simplify the updates you had to do to the dropdown. 

![Screen Shot 2021-03-08 at 10 37 51 AM](https://user-images.githubusercontent.com/10720454/110359150-62bc6f00-7ffa-11eb-8149-13899846e325.png)
![Screen Shot 2021-03-08 at 10 38 04 AM](https://user-images.githubusercontent.com/10720454/110359155-62bc6f00-7ffa-11eb-907b-e3c8ca25aa84.png)